webapp/client/web_client.jsx

```diff
@@ -95,6 +95,19 @@ class WebClientClass extends Client {
             return success(res.body);
         });
     }
+
+    samlCertificateStatus(success, error) {
+        request.get(`${this.getAdminRoute()}/saml_cert_status`).
+        set(this.defaultHeaders).
+        type('application/json').
+        accept('application/json').
+        end((err, res) => {
+            if (err) {
+                return error(err);
+            }
+            return success(res.body);
+        });
+    }
```

Comment:
We have all our other system console only APIs in the driver, including the other SAML related ones for uploading certificates I believe? Also maybe someone would like to use the driver to automate admin tasks, there's no reason not to add it to the driver. We really should only add special case stuff to web_client.jsx